#
# This patch addresses problems with the configure script.
# As it is Solaris-specific, it is not suitable for upstream.
#
--- Python-3.5.6/configure.ac	2019-01-07 04:41:11.954615334 +0000
+++ Python-3.5.6/configure.ac	2019-01-07 04:41:12.006877909 +0000
@@ -5,7 +5,7 @@ dnl ************************************
 # Set VERSION so we only need to edit in one place (i.e., here)
 m4_define(PYTHON_VERSION, 3.5)
 
-AC_PREREQ(2.65)
+AC_PREREQ(2.68)
 
 AC_INIT(python, PYTHON_VERSION, https://bugs.python.org/)
 
@@ -2507,12 +2507,6 @@ AC_MSG_CHECKING(CCSHARED)
 if test -z "$CCSHARED"
 then
 	case $ac_sys_system/$ac_sys_release in
-	SunOS*) if test "$GCC" = yes;
-		then CCSHARED="-fPIC";
-		elif test `uname -p` = sparc;
-		then CCSHARED="-xcode=pic32";
-		else CCSHARED="-Kpic";
-		fi;;
 	hp*|HP*) if test "$GCC" = yes;
 		 then CCSHARED="-fPIC";
 		 else CCSHARED="+z";
